src/Cake.Common/Tools/MSpec/MspecSettings.cs

```diff
@@ -92,14 +93,28 @@ public sealed class MSpecSettings : ToolSettings
         public bool NoTeamCity { get; set; }
 
         /// <summary>
-        ///     Gets or sets a value indicating whether to enable reporting for AppVeyor CI integration(also auto - detected)
+        ///     Gets or sets a value indicating whether to enable reporting for AppVeyor CI integration (also auto-detected).
         /// </summary>
-        public bool Appveyor { get; set; }
+        public bool AppVeyor { get; set; }
 
         /// <summary>
-        ///     Gets or sets a value indicating whether disable AppVeyor autodetection
+        ///     Gets or sets a value indicating whether to enable reporting for AppVeyor CI integration (also auto-detected).
         /// </summary>
-        public bool NoAppveyor { get; set; }
+        [Obsolete("Please use the AppVeyor property instead (note the capitalization). This property will be removed in a future version.")]
+        [CLSCompliant(false)] // Identifier differs only in case
+        public bool Appveyor { get => AppVeyor; set => AppVeyor = value; }
```

Comment:
@patriksvensson @devlead Just wanted to make sure you saw this. It looks like you have `[assembly: CLSCompliant(true)]`. Cake is C# Make, not VB Make, so I'm going to assume that CLS compliance isn't top priority for this particular property, but if that's not the case I can revert.

Reply:
So, is that important for this particular property? Should I test the behavior in a non-case-sensitive language, or just revert?